package.json

```diff
@@ -2,28 +2,29 @@
   "name": "pokemon-showdown",
   "description": "The server for the Pokémon Showdown battle simulator",
   "version": "0.11.7",
-  "main": ".sim-dist/index.js",
+  "main": "sim/index.ts",
   "dependencies": {
+    "@swc/core": "^1.2.61",
     "@types/better-sqlite3": "^5.4.0",
     "probe-image-size": "^5.0.0",
     "sockjs": "^0.3.21",
-    "sucrase": "^3.15.0"
+    "ts-node": "^10.0.0"
   },
   "optionalDependencies": {
-    "better-sqlite3": "^7.1.0",
```

Comment:
?

Are we just not using this at all anymore?

Reply:
Oh sorry `npm` screwed up my package.json and I didn't fix it properly.